Woof/Woof.xcodeproj/xcshareddata/xcschemes/Woof.xcscheme

```diff
@@ -0,0 +1,77 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Why you have included the scheme there?
Shouldn't that be a separate PR by @kirpichonok?

Reply:
sorry i' ll delete this, i'll notice @kirpichonok to create pr with scheme